boards/arm/efr32bg_sltb010a/efr32bg_sltb010a_defconfig

```diff
@@ -14,3 +14,4 @@ CONFIG_SOC_GECKO_EMU_DCDC=y
 CONFIG_SOC_GECKO_EMU_DCDC_MODE_ON=y
 CONFIG_HW_STACK_PROTECTION=y
 CONFIG_CMU_HFCLK_HFRCO=y
+CONFIG_PINCTRL=y
```

Comment:
suggestion: default PINCTRL at SoC level if it's always required see e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/gigadevice/Kconfig.defconfig#L8

Reply:
Currently BG22 has pinctrl support and it should be extended to other silabs devices in Zephyr codebase.